crates/ty_python_semantic/resources/mdtest/generics/pep695/paramspec.md

```diff
@@ -228,7 +228,9 @@ Explicit specialization of a generic class involving `ParamSpec` is done by prov
 of types, `...`, or another in-scope `ParamSpec`.
 
 ```py
+reveal_type(OnlyParamSpec[[]]().attr)  # revealed: () -> None
 reveal_type(OnlyParamSpec[[int, str]]().attr)  # revealed: (int, str, /) -> None
+reveal_type(OnlyParamSpec[(int, str)]().attr)  # revealed: (int, str, /) -> None
```

Comment:
same here, you could add a comment explaining why we support this even though it's "highly distasteful" 😆